docs/blog/newsletter-december-2017.rst

```diff
@@ -0,0 +1,90 @@
+.. post:: December 18, 2017
+   :tags: newsletter
+
+
+#########################################
+Write the Docs Newsletter – December 2017
+#########################################
+
+Hi everybody! Kelly O. here, with our last issue of 2017! It's been a gigantic year for the Write the Docs community. Just in the last month, our members have gotten up to some amazing stuff, including:
+* a very successful first Australian event, which you can watch videos from `here <https://www.youtube.com/channel/UCPhWNEFb53x6PjnpRIYf1yg/videos>`_ and read about `here <https://ffeathers.wordpress.com/2017/11/25/doc-sprint-at-write-the-docs-day-australia/>`_.
```

Comment:
I generally recommend 

>  which you can `watch videos 
 <https://www.youtube.com/channel/UCPhWNEFb53x6PjnpRIYf1yg/videos>`_ from and `read about  <https://ffeathers.wordpress.com/2017/11/25/doc-sprint-at-write-the-docs-day-australia/>`_.

instead of linking **here**

Reply:
Thoughout .... :-p

Reply:
👍